docs/features.md

```diff
@@ -209,10 +209,75 @@ Make Julia start listening on a given port and return lint messages to requests
 This feature is useful when you want to lint julia code in a non julia environment.
 
 Existing plugins:
+
  * Sublime Text: [SublimeLinter-contrib-julialintserver](https://github.com/invenia/SublimeLinter-contrib-julialintserver)
  * linter-julia for Atom: [linter-julia](https://github.com/TeroFrondelius/linter-julia)
 
-The protocol for the server is:
+The new protocol for the server is JSON in both input and output:
+```json
+"{
+    \"file\":\"path_to_the_file\",
```

Comment:
This is a Julia string that happens to contain JSON, not JSON. We should not have the `\"` and the surrounding `""` in the README.